libraries/botbuilder-core/tests/telemetryMiddleware.test.js

```diff
@@ -511,32 +559,36 @@ describe(`TelemetryMiddleware`, function () {
             .send('foo')
             .delay(100)
             .send('update')
-            .delay(100)
-            .then(done());
+            .delay(100);
     });
 
-    it(`telemetry should log channel specific properties`, function (done) {
+    it(`telemetry should log channel specific properties`, function(done) {
```

Comment:
This is the test that was actually failing, uncaught. The error was that it hit the `assert(false)` on removed line 536. This is because it gets called a 2nd time. I didn't account for the second call because, as far as I can tell, we're only trying to verify the assert information in `case 1:`